lib/web/scripts.go

```diff
@@ -0,0 +1,149 @@
+	}
+
+	// if there's a rollout, we do new autoupdates
+	_, rolloutErr := h.cfg.AccessPoint.GetAutoUpdateAgentRollout(ctx)
+	if rolloutErr != nil && !trace.IsNotFound(rolloutErr) {
+		h.logger.WarnContext(ctx, "Failed to get rollout", "error", rolloutErr)
+		return scripts.InstallScriptOptions{}, trace.Wrap(err, "failed to check the autoupdate agent rollout state")
+	}
+
+	var autoupdateStyle scripts.AutoupdateStyle
+	switch {
+	case rolloutErr == nil:
+		autoupdateStyle = scripts.UpdaterBinaryAutoupdate
+	case automaticUpgrades(h.clusterFeatures):
+		autoupdateStyle = scripts.PackageManagerAutoupdate
+	default:
+		autoupdateStyle = scripts.NoAutoupdate
+	}
+
+	var teleportFlavor string
+	switch modules.GetModules().BuildType() {
+	case modules.BuildEnterprise:
+		teleportFlavor = types.PackageNameEnt
+	case modules.BuildOSS, modules.BuildCommunity:
+		teleportFlavor = types.PackageNameOSS
+	default:
+		h.logger.WarnContext(ctx, "Unknown built type, defaulting to the 'teleport' package.", "type", modules.GetModules().BuildType())
+		teleportFlavor = types.PackageNameOSS
+	}
+
+	cdnBaseURL, err := getCDNBaseURL()
+	if err != nil {
+		h.logger.WarnContext(ctx, "Failed to get CDN base URL", "error", err)
+		return scripts.InstallScriptOptions{}, trace.Wrap(err)
+	}
+
+	return scripts.InstallScriptOptions{
+		AutoupdateStyle: autoupdateStyle,
+		TeleportVersion: version,
+		// Note: this override is required to configure the license on AGPL builds.
+		// We cannot install random binaries if the user doesn't
+		CDNBaseURL:     cdnBaseURL,
+		ProxyAddr:      h.PublicProxyAddr(),
+		TeleportFlavor: teleportFlavor,
+		FIPS:           modules.IsBoringBinary(),
+	}, nil
+
+}
+
+// EnvVarCDNBaseURL is the environment variable that allows users to override the Teleport base CDN url used in the installation script.
+// Setting this value is required for testing (make production builds install from the dev CDN, and vice versa).
+// As we (the Teleport company) don't distribute AGPL binaries, this must be set when using a Teleport OSS build.
+// Example values:
+// - "https://cdn.teleport.dev" (prod)
+// - "https://cdn.cloud.gravitational.io" (dev builds/staging)
+const EnvVarCDNBaseURL = "TELEPORT_CDN_BASE_URL"
+
+func getCDNBaseURL() (string, error) {
+	// If the user explicitly overrides the CDN base URL, we use it.
+	if override := os.Getenv(EnvVarCDNBaseURL); override != "" {
+		return override, nil
+	}
+
+	// If this is an AGPL build, we don't want to automatically install binaries distributed under a more restrictive
+	// license so we error and ask the user set the CDN URL, either to:
+	// - the official Teleport CDN if they agree with the community license and meet its requirements
+	// - a custom CDN where they can store their own AGPL binaries
+	if modules.GetModules().BuildType() == modules.BuildOSS {
+		return "", trace.BadParameter(
+			"This proxy licensed under AGPL but CDN binaries are licensed under the more restrictive Community license. "+
```

Comment:
```suggestion
			"This proxy is licensed under AGPL but CDN binaries are licensed under the more restrictive Community Edition license. "+
```

IANAL, but I think "Community Edition" is the naming we should use (it's the one that appears in multiple Teleport user facing pages).